src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.ts

```diff
@@ -846,7 +849,8 @@ export class OnlineGameWrapperComponent extends GameWrapper<MinimalUser> impleme
             if (this.opponentSubscription.isPresent()) {
                 this.opponentSubscription.get()();
             }
-            this.gameService.stopObserving();
+            // Unsubscribe from the part
+            this.partUnsubscribe.get()();
```

Comment:
j'suis pas sûr d'être convaincu par les ()() :D

Reply:
une meilleure proposition ? `const actualPartUnsubscribe = this.partUnsubscribe.get(); actualPartUnsubscribe();` ? C'pas fou non plus lo

Reply:
non mais pas retourner de () => void quoi :/ retourner du { unsubscribe: () => void } peut être :D ?

Reply:
mais c'est pas nous qui retournons du () => void, c'est firestore ! Après on pourrait les englober oui, je fais ça ! (TODO)